services/movie.py

```diff
@@ -1,11 +1,13 @@
 from django.db.models import QuerySet
+from django.db import transaction
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Using `list[int] = None` as a default argument is not recommended because lists are mutable. Consider using `Optional[List[int]] = None` from the `typing` module to avoid potential issues.